v2.1/detailed-sql-support.md

```diff
@@ -7,682 +7,682 @@ This page lists which SQL standard features are supported, partially-supported,
+X135 | XMLBINARY clause in subqueries | No
+X141 | IS VALID predicate: data-driven case | No
+X142 | IS VALID predicate: ACCORDING TO clause | No
+X143 | IS VALID predicate: ELEMENT clause | No
+X144 | IS VALID predicate: schema location | No
+X145 | IS VALID predicate outside check constraints | No
+X151 | IS VALID predicate with DOCUMENT option | No
+X152 | IS VALID predicate with CONTENT option | No
+X153 | IS VALID predicate with SEQUENCE option | No
+X155 | IS VALID predicate: NAMESPACE without ELEMENT clause | No
+X157 | IS VALID predicate: NO NAMESPACE with ELEMENT clause | No
+X160 | Basic Information Schema for registered XML Schemas | No
+X161 | Advanced Information Schema for registered XML Schemas | No
+X170 | XML null handling options | No
+X171 | NIL ON NO CONTENT option | No
+X181 | XML(DOCUMENT(UNTYPED)) type | No
+X182 | XML(DOCUMENT(ANY)) type | No
+X190 | XML(SEQUENCE) type | No
+X191 | XML(DOCUMENT(XMLSCHEMA)) type | No
+X192 | XML(CONTENT(XMLSCHEMA)) type | No
+X200 | XMLQuery | No
+X201 | XMLQuery: RETURNING CONTENT | No
+X202 | XMLQuery: RETURNING SEQUENCE | No
+X203 | XMLQuery: passing a context item | No
+X204 | XMLQuery: initializing an XQuery variable | No
+X205 | XMLQuery: EMPTY ON EMPTY option | No
+X206 | XMLQuery: NULL ON EMPTY option | No
+X211 | XML 1.1 support | No
+X221 | XML passing mechanism BY VALUE | No
+X222 | XML passing mechanism BY REF | No
+X231 | XML(CONTENT(UNTYPED)) type | No
+X232 | XML(CONTENT(ANY)) type | No
+X241 | RETURNING CONTENT in XML publishing | No
+X242 | RETURNING SEQUENCE in XML publishing | No
+X251 | Persistent XML values of XML(DOCUMENT(UNTYPED)) type | No
+X252 | Persistent XML values of XML(DOCUMENT(ANY)) type | No
+X253 | Persistent XML values of XML(CONTENT(UNTYPED)) type | No
+X254 | Persistent XML values of XML(CONTENT(ANY)) type | No
+X255 | Persistent XML values of XML(SEQUENCE) type | No
+X256 | Persistent XML values of XML(DOCUMENT(XMLSCHEMA)) type | No
+X257 | Persistent XML values of XML(CONTENT(XMLSCHEMA)) type | No
+X260 | XML type: ELEMENT clause | No
+X261 | XML type: NAMESPACE without ELEMENT clause | No
+X263 | XML type: NO NAMESPACE with ELEMENT clause | No
+X264 | XML type: schema location | No
+X271 | XMLValidate: data-driven case | No
+X272 | XMLValidate: ACCORDING TO clause | No
+X273 | XMLValidate: ELEMENT clause | No
+X274 | XMLValidate: schema location | No
+X281 | XMLValidate with DOCUMENT option | No
+X282 | XMLValidate with CONTENT option | No
+X283 | XMLValidate with SEQUENCE option | No
+X284 | XMLValidate: NAMESPACE without ELEMENT clause | No
+X286 | XMLValidate: NO NAMESPACE with ELEMENT clause | No
+X300 | XMLTable | No
+X301 | XMLTable: derived column list option | No
+X302 | XMLTable: ordinality column option | No
+X303 | XMLTable: column default option | No
+X304 | XMLTable: passing a context item | No
+X305 | XMLTable: initializing an XQuery variable | No
+X400 | Name and identifier mapping | No
+X410 | Alter column data type: XML type | No
```

Comment:
Can this change be limited to only removing the bold formatting from the table rows as requested in the linked issue?  I totes understand the philosophy of "let's fix this while we're in here" but I think a formatting-only change of this size might be better considered in a separate PR.  This diff is huge and while it's probably fine I can't honestly say I'm able to give it a thorough review, and I wouldn't want to miss something obvious.